lld/ELF/SyntheticSections.cpp

```diff
@@ -1055,74 +1059,122 @@ void MipsGotSection::build() {
     ctx.symAux.back().gotIdx = p.second;
   }
 
-  // Create dynamic relocations.
+  // Create relocations.
+  //
+  // NB: GOT 'page address' entries have their VAs handled in writeTo as they
+  // reference an OutputSection not a Symbol.
   for (FileGot &got : gots) {
-    // Create dynamic relocations for TLS entries.
+    static Undefined dummy(ctx.internalFile, "", STB_LOCAL, 0, 0);
```

Comment:
NB: There are now three instances of this (one non-static, which I guess is more correct with the ctx-ification of LLD...). Perhaps ctx should grow this as a member that's constructed at the same time as internalFile?